src/chocolatey/infrastructure.app/commands/ChocolateyUpgradeCommand.cs

```diff
@@ -63,6 +63,9 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                 .Add("params=|parameters=|pkgparameters=|packageparameters=|package-parameters=",
                      "PackageParameters - Parameters to pass to the package. Defaults to unspecified.",
                      option => configuration.PackageParameters = option.remove_surrounding_quotes())
+                .Add("applyToDependentPackages",

```

Comment:
This should be all lowercase here. Note this value doesn't tell me independently by the name what it is applying to dependent packages. It should probably be `applyparameterstodependentpackages` or something similar.